frontend/src/components/Send.js

```diff
@@ -18,16 +22,26 @@ Send.propTypes = {
   available: PropTypes.number.isRequired,
   authzTx: PropTypes.object.isRequired,
   networkName: PropTypes.string.isRequired,
+  authzMode: PropTypes.bool.isRequired,
+  grantsToMe: PropTypes.array.isRequired,
 };
 
 export default function Send(props) {
-  const { chainInfo, sendTx, available, onSend, authzTx } = props;
+  const {
+    chainInfo,
+    sendTx,
+    available,
+    onSend,
+    authzTx,
+    isAuthzMode,
+    grantsToMe,
+    setGranter,
+    granter
+  } = props;
 
-  const params = useParams();
-
-  const selectNetwork = useSelector(
-    (state) => state.common.selectedNetwork.chainName
-  );
+  // const [granter, setGranter] = React.useState(
```

Comment:
Why this code is commented?

Reply:
We are passing granter and setGranter as props to Send.js.
I will remove these comments.